src/app/custom-components/event-location/event-location.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component } from "@angular/core";
+import { IMapOptions, IMarkerIconInfo, ILatLong } from "angular-maps";
+
+const LATITUDE: number = -17.41809191207568;
+const LONGITUDE: number = -66.12963527113635;
```

Comment:
It's not necessary to specify the type "number", since this value is a constant and is already defined. Tslint should have marked this as an error.